app/services/nomis/api.rb

```diff
@@ -48,7 +48,11 @@ def lookup_offender_location(noms_id:)
         client.get("/offenders/#{noms_id}/location")
       }
 
-      Nomis::Establishment.new(response['establishment']).tap do |establishment|
+      attributes = response['establishment']
```

Comment:
I think is cleaner and easier to test to push this logic to a class method on `Nomis::Establishment`, like we do on the [Booking class](https://github.com/ministryofjustice/prison-visits-2/blob/e891d6f726e908f7d844015d825c21698fb890a8/app/services/nomis/booking.rb#L8)